packages/story-editor/src/app/media/local/reducers/setMediaType.js

```diff
@@ -24,7 +28,7 @@ function setMediaType(state, { mediaType }) {
   }
   return {
     ...INITIAL_STATE,
-    media: state.media.filter(({ local }) => local), // This filter allows remove temporary file returned on upload
+    media: state.media.filter(({ src }) => isBlobURL(src)), // This filter allows remove temporary file returned on upload
```

Comment:
Yeah but to filter out local resources you'd do `state.media.filter(({ local }) => ! local)` and not `state.media.filter(({ local }) => local)` 

Again, this would _keep_ the items, not _remove_ them, which seems odd.

Am I missing something?